hack/check_tuningPolicy.sh

```diff
@@ -45,4 +45,22 @@ for jpath in "${RATE_LIMITS[@]}"; do
      exit 1
   fi
   sleep 2
-done
\ No newline at end of file
+done
+
+echo "Check that the TuningPolicy highBurst is configuring the KV object as expected"
+
+./hack/retry.sh 10 3 "(${KUBECTL_BINARY} patch -n \"${INSTALLED_NAMESPACE}\" hco kubevirt-hyperconverged --type=json -p='[{"op": "replace", "path": "/spec/tuningPolicy", "value": "highBurst"}]')"
```

Comment:
I think it's failing because of the new check we added. The annotation is still there.